Control coverage/Feature/LogicApps.md

```diff
@@ -124,3 +126,126 @@ Using the firewall feature ensures that access to the data or the service is res
+
+### Control Spec
+
+> **Passed:**
+> If any of the following conditions are met:
+>   <br />a. All connectors in Logic App using AAD Auth ( "ActiveDirectoryOAuth", "ManagedServiceIdentity").
+>   <br />b. No such connector found in resource which is currently being evaluated by AzTS.
+> 
+> **Failed:**
+> One or more connector in resource is not using AAD Auth.
+> 
+
+### Recommendation
+
+- **Azure Portal**
+
+  For HTTP based connectors, Go to Azure Portal --> Logic App --> Logic app designer --> For each non compliant connector --> Update Authentication type to either Managed Identity or Active Directory Oauth. For more details on AAD auth, refer: https://docs.microsoft.com/en-us/azure/connectors/connectors-native-http#azure-active-directory-oauth-authentication. For other connectors you must manually verify that AAD authentication is used for connectors that support it.
+
+<!--
+- **PowerShell**
+
+	```powershell
+	Set-AzStorageContainerAcl -Name '<ContainerName>' -Permission 'Off' -Context (New-AzStorageContext -StorageAccountName '<StorageAccountName>' -StorageAccountKey '<StorageAccountKey>')
+	```
+
+	For more help:
+	```powershell
+	Get-Help Set-AzStorageContainerAcl -full
+	```
+-->
+
+<!--
+- **Enforcement Policy**
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/View_Definition.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>)
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>)
+-->
+
+### Azure Policy or ARM API used for evaluation
+
+- ARM API to list Logic apps at subscription level: 
+/subscriptions/{subscriptionId}/providers/Microsoft.Logic/workflows?api-version=2016-06-01<br />
+**Properties:** [\*].properties.definition.actions[*].type, [\*].properties.definition.actions[*].inputs, [\*].properties.parameters.$connections.value[*].connectionId <br />
+
+- ARM API to list API Connections at subscription level:
+/subscriptions/{subscriptionId}/providers/Microsoft.Web/connections?api-version=2016-06-01<br />
+**Properties:** [\*].properties.api.name
+<br />
+<br />
+
+___
+
+## Azure_LogicApps_DP_Connectors_Encrypt_Data_In_Transit
+
+### Display Name
+Data transit across Logic App connectors must use encrypted channel
+
+### Rationale
+Use of HTTPS ensures server/service authentication and protects data in transit from network layer man-in-the-middle, eavesdropping, session-hijacking attacks.
+
+### Control Spec
+
+> **Passed:**
+> If any of the following conditions are met:
+>   <br />a. All evaluated connector(s) in resource are using HTTPS URI.
+>   <br />b. No such connector found in resource which is currently being evaluated by AzTS.
```

Comment:
Same comment as above, We'll need to publish the list of connectors currently being evaluated